pkgs/development/node-packages/node-env.nix

```diff
@@ -363,7 +363,7 @@ let
 
         npm ${forceOfflineFlag} --nodedir=${nodeSources} ${npmFlags} ${stdenv.lib.optionalString production "--production"} rebuild
 
-        if [ "''${dontNpmInstall-}" != "1" ]
```

Comment:
This reverts treewide patch: 2811b032d639684d3983282e9d10d89f37d161d5

node2nix should be patched to include that change to avoid generating that diff on each run.

Reply:
This has been reported upstream in https://github.com/svanderburg/node2nix/issues/162.